src/cart_tumor.cc

```diff
@@ -80,7 +82,8 @@ int Simulate(int argc, const char** argv) {
       std::make_unique<DiffusionThomasAlgorithm>(
           kOxygen, "oxygen", kDiffusionCoefficientOxygen, kDecayConstantOxygen,
           kResolutionGridSubstances, kDtSubstances,
-          true);  // true indicates Dirichlet border conditions
+          // true indicates Dirichlet border conditions
+          true);
```

Comment:
```suggestion
          /*dirichlet_border=*/true);
```

That's the syntax we use in these cases.